NativeYoutube/Views/SharedViews/VideoRowView.swift

```diff
@@ -19,22 +19,22 @@ struct VideoRowView: View {
                 WebImage(url: video.thumbnail)
                     .resizable()
                     .overlay {
-                        Rectangle()
+                        RoundedRectangle(cornerRadius: 5)
                             .fill(hovered ? .ultraThinMaterial : .ultraThickMaterial)
                     }
 
-                HStack {
-                    if !hovered {
-                        WebImage(url: video.thumbnail)
-                            .resizable()
-                            .scaledToFill()
-                            .frame(width: 128, height: 72)
-                            .cornerRadius(5)
-                            .shadow(radius: 6, x: 2)
-                            .padding(.leading, 4)
-                            .padding(.vertical, 2)
-                            .transition(.offset(x: -128))
-                    }
+                HStack() {
+                    WebImage(url: video.thumbnail)
+                        .resizable()
+                        .scaledToFill()
+                    // When hovered, reduce size of the thumbnail...
+                        .frame(width: hovered ? 64 : 128, height: hovered ? 36 : 72)
+                        .cornerRadius(5)
+                        .shadow(radius: 6, x: 2)
+                    // ...and move it a bit on the left to avoid looking weird.
```

Comment:
I'm curious why the thumbnail was resized instead of "hiding" it.

Wouldn't the ability to see more title be better?

Reply:
Infact, i found that having the thumbnail disappearing when you hover was really weird, so i tried to make it smaller instead. I havent seen any video title that wasn't displayed entirely on hover so i kept it like that

Also, as thumbnails in Youtube are really important, i feel like you should be able to click on the thumbnail to play the video, and not having it disappear right when you wanna click on it :0

I'm sorry if i'm unclear but yeah.. i really feel like it's frustrating to not being able to catch the thumbnail with the mouse LOL (but ofc it's a design choice so i understand if you prefer hiding it :3)

Reply:
😅 definitely a design choice!

Since we can see the thumbnails before we hover on the video I think it's safe to hide them!

Ideally we'd animate the thumbnail YouTube style so it shows the preview of the video but I'll need to take a look at YouTube api for that!

> Let's slide them out for now.

I'll consider adding a themes tab on settings so users can toggle between them and maybe even customize the app a little bit!